packages/plugin-ext/doc/how-to-add-new-plugin-namespace.md

```diff
@@ -90,23 +122,59 @@ function findPlugin(filePath: string): Plugin | undefined {
 }
 ```
 
-## Next you need to implement `createAPIFactory` factory function
+## Implement your API object
```

Comment:
The `findPlugin()` implementation at line 120 (can't add a comment there) doesn't work when the plugin is found via a symlink, which is typically the scenario during development. See [12840](https://github.com/eclipse-theia/theia/issues/12840) for a more detailed explanation of the problem and how to address it.

Reply:
I don't think that this problem exists anymore as the issue that you referenced was actually closed and solved outside the external plugin API by ensuring that the `manifest.packagePath` already has a properly resolved path.

Reply:
Ahhh. You know, I somehow missed all the activity after the initial commit/fix was proposed in that pull request--the one that relied on `fs.realpathSync`! I had no idea the solution was reworked and the fix moved elsewhere in two subsequent commits. I still have the original fix in my product...lol. Guess I can remove that!